frontend/webadmin/modules/webadmin/src/main/java/org/ovirt/engine/ui/webadmin/section/main/view/popup/storage/RegisterEntityInfoPanel.java

```diff
@@ -312,8 +312,13 @@ protected Double getSpeed(VmNetworkInterface object) {
         AbstractTextColumn<VmNetworkInterface> dropsColumn = new AbstractSumUpColumn<VmNetworkInterface>() {
             @Override
             protected Double[] getRawValue(VmNetworkInterface object) {
-                Double receiveDrops = object != null ? object.getStatistics().getReceiveDrops().doubleValue() : null;
-                Double transmitDrops = object != null ? object.getStatistics().getTransmitDrops().doubleValue() : null;
+                Double receiveDrops = (object != null && object.getStatistics() != null
+                               && object.getStatistics().getReceiveDrops() != null)
+                        ? object.getStatistics().getReceiveDrops().doubleValue() : 0.0;
+                Double transmitDrops = (object != null &&  object.getStatistics() != null
```

Comment:
Double whitespace :)

Reply:
Thanks. Removed the double space.

Reply:
The double space between `&&  object.getStatistics()` is still there? :)

Reply:
If you are referring to line 318, I do not see double space after my update.